File: triangulo/app/src/main/java/ddm/tads/ifrn/br/triangulo/MainActivity.java
package ddm.tads.ifrn.br.triangulo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button btnCalcular;
    private EditText editBase, editAltura, editArea;

    private View.OnClickListener listener = new View.OnClickListener() {
        @Override
        public void onClick(View v) {
            int base = Integer.parseInt(editBase.getText().toString());
            int altura = Integer.parseInt(editAltura.getText().toString());
            double area = (base * altura)/2d;
            editArea.setText("" + area);
            editBase.setText("");
            editAltura.setText("");
        }
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnCalcular = (Button) findViewById(R.id.btnCalcular);
        editBase = (EditText) findViewById(R.id.editBase);
        editAltura = (EditText) findViewById(R.id.editAltura);
        editArea = (EditText) findViewById(R.id.editArea);

        btnCalcular.setOnClickListener(listener);

    }


}

File: IMCKotlin2/app/src/main/java/br/edu/ifrn/tads/movies/imckotlin/MainActivity.kt
package br.edu.ifrn.tads.movies.imckotlin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        calc_btn.setOnClickListener{CalculaIMC()}
    }

    private fun CalculaIMC(){
        val altura = editAltura.text.toString().toDouble()
        val peso = editPeso.text.toString().toDouble()
        val imc = (peso / (altura * altura)).toString()
        val intent = Intent(this, ResultadoActivity::class.java)
        intent.putExtra("IMC", imc)
        startActivity(intent)
    }
}
